geopandas/tests/test_extension_array.py

```diff
@@ -287,7 +287,20 @@ def test_registry(self, data, dtype):
 
 
 class TestInterface(extension_tests.BaseInterfaceTests):
-    pass
+    def test_contains(self, data, data_missing):
+        # overrided due to the inconsistency between
+        # GeometryDtype.na_value = np.nan
+        # and None being used as NA in array
+
+        # ensure data without missing values
+        data = data[~data.isna()]
+
+        # first elements are non-missing
+        assert data[0] in data
+        assert data_missing[0] in data_missing
+
+        assert None in data_missing
+        assert None not in data
```

Comment:
```suggestion
        assert None not in data
        assert pd.NaT not in data_missing
```

(codecov complained (correctly) about a non-covered line)